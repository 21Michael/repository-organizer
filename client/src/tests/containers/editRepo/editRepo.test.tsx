import React from "react";
import { render, cleanup } from "@testing-library/react";
import EditRepo from "../../../containers/editRepo/editRepo";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("EditRepo: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("editRepo", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <EditRepo />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
