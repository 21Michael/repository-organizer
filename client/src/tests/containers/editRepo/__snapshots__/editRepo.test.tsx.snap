// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditRepo:  editRepo 1`] = `
<div>
  <div
    class="wrapper"
  >
    <form
      class="form"
    >
      <h1
        class="title"
      >
         
        Edit a repository
         
      </h1>
      <div
        class="wrapper"
      >
        <label
          class="label"
        >
          Name
          :
          <div
            class="wrapper"
          >
            <input
              class="input"
              name="name"
              type="text"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
        <label
          class="label"
        >
          Description
          :
          <div
            class="wrapper"
          >
            <textarea
              class="input"
              name="description"
              rows="7"
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
        <label
          class="label"
        >
          Stars
          :
          <div
            class="wrapper"
          >
            <input
              class="input"
              name="stars"
              type="number"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
        <label
          class="label"
        >
          Creator Name
          :
          <div
            class="wrapper"
          >
            <input
              class="input"
              name="creatorName"
              type="text"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
        <label
          class="label"
        >
          Created At
          :
          <div
            class="wrapper"
          >
            <input
              class="input"
              name="createdAt"
              type="date"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
      </div>
      <div
        class="wrapper"
      >
        <button
          class="button null"
          name="submit"
          type="submit"
        >
          Save
        </button>
      </div>
    </form>
  </div>
</div>
`;
